docs/standalone/audit_guidelines.md

```diff
@@ -0,0 +1,137 @@
+To whomever decides to perform an assessment of the security of the Hydra Head protocol implementation version 0.9.
+
+Hydra is an open-source project and, as such, can be freely used, reviewd and audited. Shall you want to perform a
+security audit, the hydra team would suggest to apply static and dynamic analysis, focused, but not limited to, the areas of concern below.
+
+Shall you decide to share your findings with us, please consider the following, as Per [CIP-52](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0052):
+1. Any discrepancies, deviations or spotted vulnerabilities shall be described and classified with an appropriate severity level. Recommendations to rectify the identified deficiencies shall also be provided whenever appropriate.
+2. When automated tools are used as a replacement for manual review/code inspection, they shall be documented or referenced. Note that it’s the responsibility of the auditor to ensure that such tooling may not exhibit potential failures that can adversely affect the review outcome.
+3. Any strategies/methodologies used to assess the consistency, correctness and completeness of the requirements shall also be documented or referenced.
```

Comment:
Maybe we can add a note on how to get in touch should one want to do one and how to safely disclose any findings. Maybe we could refer to this https://github.com/input-output-hk/cardano-engineering-handbook/tree/main/policy/security#security.. or just be friendly and accept messages on any of our communication channels from https://github.com/input-output-hk/hydra/blob/master/SUPPORT.md.

Reply:
I also though it was missing and agree that is something that we should do.

I'm not sure the engineering handbook is adapted to a public audience.

You're right that our SUPPORT page should provide a _responsible disclosure_ process. But I don't think we have defined any yet.

Let's see if we can find something easy and actionable before merging this P.R. or update this document later if it appears it takes too much time.